corehq/apps/locations/resources/v0_1.py

```diff
@@ -40,18 +40,22 @@ def all_ids():
     if user.is_domain_admin(project.name):
         return all_ids()
 
-    user_loc = (user.get_location(project.name) if isinstance(user, WebUser)
-                else user.location)
-    if not user_loc:
+    user_locs = user.get_sql_locations(project.name)
+
+    if not user_locs:
         return all_ids()
 
-    editable = list(user_loc.sql_location.get_descendants(include_self=True)
-                    .values_list('location_id', flat=True))
+    editable = []
+    for user_loc in user_locs:
+        editable.extend(
+            list(user_loc.sql_location.get_descendants(include_self=True).values_list('location_id', flat=True))
```

Comment:
Seems like this could balloon to a large amount of locations for some users on ICDS. (essentially every location in the tree)